tkn logs segfaults if run not started

# Expected Behavior

Some info/error/warning, notifying me the job has not started yet

# Actual Behavior

```
$ tkn pipelinerun list
NAME                              STARTED        DURATION     STATUS      
....
nextcloud-wsweet-master-1         ---            ---          ---         
...
$ tkn pipelinerun logs nextcloud-wsweet-master-1
panic: runtime error: index out of range [0] with length 0

goroutine 34 [running]:
github.com/tektoncd/cli/pkg/cmd/pipelinerun.(*LogReader).readAvailableLogs.func1(0xc000772000, 0xc000772060, 0x23cb680, 0x0, 0x0, 0xc000200720, 0xc0000e0a00)
	/workspace/src/github.com/tektoncd/cli/pkg/cmd/pipelinerun/log_reader.go:137 +0x3a5
created by github.com/tektoncd/cli/pkg/cmd/pipelinerun.(*LogReader).readAvailableLogs
	/workspace/src/github.com/tektoncd/cli/pkg/cmd/pipelinerun/log_reader.go:126 +0x2e1
```

# Steps to Reproduce the Problem

1. pile up several pipelineruns
2. use tkn pipelinerun logs against a job that did not start

--- a/pkg/cmd/pipelinerun/log_reader.go
+++ b/pkg/cmd/pipelinerun/log_reader.go
@@ -95,10 +95,8 @@ func (lr *LogReader) readLiveLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-chan 
 
 		wg.Wait()
 
-		if pr.Status.Conditions != nil {
-			if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
-				errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
-			}
+		if !empty(pr.Status) && pr.Status.Conditions[0].Status == corev1.ConditionFalse {
+			errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
 		}
 	}()
 
@@ -134,7 +132,8 @@ func (lr *LogReader) readAvailableLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-
 
 			pipeLogs(logC, errC, tlr)
 		}
-		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
+
+		if !empty(pr.Status) && pr.Status.Conditions[0].Status == corev1.ConditionFalse {
 			errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
 		}
 	}()
@@ -213,5 +212,10 @@ func pipeLogs(logC chan<- Log, errC chan<- error, tlr *taskrun.LogReader) {
 }
 
 func empty(status v1alpha1.PipelineRunStatus) bool {
+
+	if status.Conditions == nil {
+		return true
+	}
+
 	return len(status.Conditions) == 0
 }

